specification/reservations/resource-manager/Microsoft.Capacity/stable/2022-11-01/examples/Return.json

```diff
@@ -15,9 +15,33 @@
     }
   },
   "responses": {
+    "200": {
+      "body": {
+        "id": "/providers/microsoft.capacity/reservationOrders/50000000-aaaa-bbbb-cccc-100000000004",
+        "type": "Microsoft.Capacity/reservationOrders",
+        "name": "50000000-aaaa-bbbb-cccc-100000000004",
+        "etag": 7,
+        "properties": {
+          "displayName": "cabri",
+          "requestDateTime": "2017-08-30T03:49:19.0252126Z",
+          "createdDateTime": "2017-08-30T03:51:49.8083758Z",
+          "expiryDate": "2018-08-30",
```

Comment:
We are not returning expiryDate in getOrder api I believe, can you remove this?

Reply:
We are now adding in the response